src/docs/topics/red-hat-ecosystem-catalog-certification/README.md

```diff
@@ -0,0 +1,34 @@
+# Managing the Orchard Core Red Hat Ecosystem Catalog certification
+
+The [Red Hat Ecosystem Catalog](https://catalog.redhat.com/) is a catalog of software certified to work on the Red Hat platforms, like directly on Red Hat Enterprise Linux (RHEL). Orchard Core is cross-platform and thus works on RHEL as well. To showcase this, and to publicize Orchard Core better to the Red Hat community, we have certified Orchard and created [its Red Hat Ecosystem Catalog profile](https://catalog.redhat.com/software/applications/detail/223797).
+
+## Renewing the certification
+
+The certification is an assurance that Orchard Core works on RHEL. To keep this certification up-to-date, we need to rerun certification for each new Orchard release, as well as RHEL major release. This is not mandatory though, and it's good enough if we do this certification occasionally.
+
+Note that Orchard has to be certified as a non-containerized app. Containerized app certification can't be done with [the Orchard Core Docker image](https://hub.docker.com/r/orchardproject/orchardcore-cms-linux), since Red Hat required Podman.
```

Comment:
```suggestion
Note that Orchard has to be certified as a non-containerized app. Containerized app certification can't be done with [the Orchard Core Docker image](https://hub.docker.com/r/orchardproject/orchardcore-cms-linux), since Red Hat required [Podman](https://podman.io).
```